### PR TITLE
Remove dependency on GeoPHP module. Use composer dependencies instead

### PR DESCRIPTION
**Pull request by plopesc**
GeoPHP module should not be required anymore. Dependencies can be managed using composer.

Thanks!

### CODE REVIEW

**Comment by kostajh** [comment]
Thanks @plopesc, will review soon!

File: composer.json [hunk 1]
```diff
@@ -2,9 +2,12 @@
     "name": "drupal/views_geojson",
     "version": "master",
     "type": "drupal-module",
-    "soure": {
+    "source": {
```
Comment (kostajh) [inline composer.json]: @plopesc `composer install` complains that `reference` isn't defined. We could probably remove the `source` key entirely?

Comment (kostajh) [inline composer.json]: How's this as a new `composer.json`?

``` json
{
  "name": "drupal/views_geojson",
  "version": "master",
  "description": "Provides a GeoJSON plugin to Views",
  "type": "drupal-module",
  "license": "GPL-2.0+",
  "keywords": ["views", "geojson"],
  "homepage": "https://www.drupal.org/project/views_geojson",
  "minimum-stability": "dev",
  "require": {
    "phayes/geophp": "dev-master"
  }
}
```

File: views_geojson.module [hunk 2]
```diff
@@ -81,8 +81,7 @@ function _views_geojson_render_fields($view, $row, $index) {
         }
       }
       if (!empty($wkt)) {
-        geophp_load();
-        $json = geoPHP::load($wkt, 'wkt');
+        $json = \geoPHP::load($wkt, 'wkt');
         if (is_object($json)) {
```
Comment (kostajh) [inline views_geojson.module]: @plopesc haven't tested yet, but shouldn't this follow the same style as above? e.g. set `$geophp = \Drupal::getContainer()->get('geofield.geophp');`, then call `$geophp->load()`

File: composer.json (hunk 1 above)
Comment (plopesc) [inline composer.json]: Looks good to me

File: views_geojson.module (hunk 2 above)
Comment (plopesc) [inline views_geojson.module]: @kostajh Nope, here we have 2 different use cases.
In the first one we're processing a Geofield item and then, we know that Geofield module is enabled, so we use the service provided by this module. By default that service is GeoPHP library, but can change.
On the other hand we are processing a generic WKT string that can be stored in a regular text field. So we don't know if Geofield module is present or not. That's why we use directly the GeoPHP library, which is required by views_geojson module.

Comment (kostajh) [inline views_geojson.module]: makes sense, thanks!

**Comment by plopesc** [comment]
Thanks!
When this module will be available in drupal.org?
I'm using it and works fine

**Comment by kostajh** [comment]
@plopesc could you post a comment here letting the maintainer know that it's working for you? I'd like to get an 8.x-1.x branch opened up on Drupal.org and maintain the project over there https://www.drupal.org/node/2527636

**Pull request merged**